lib/DynamicGc.mli

```diff
@@ -32,6 +32,13 @@ type config = {
   heap_really_worry_mb : int;
 }
 
+module Config : sig
+  (* Produces a simple config that will set space_overhead to 120, the default,
+   * when the size of the major heap is below the given threshold. When the heap
+   * is larger than the given threshold, space_overhead will be set to 80. *)
```

Comment:
ocamldoc?

Reply:
Sure, although I don't think ocamldoc will do a whole lot for us until/unless we start using it properly and consistently.

Reply:
I think it is more important on external-facing libs.